### ISSUE TITLE
[Windows] 

### ISSUE THREAD

**Issue by cbracken** [flutter/flutter#109804]
Internal: 241886719

```dart
Scaffold(
      appBar: AppBar(
        // Here we take the value from the MyHomePage object that was created by
        // the App.build method, and use it to set our appbar title.
        // We also apply a custom font, ProductSans.
        title: Text(widget.title),
      ),
      body: Column(
        children: [
          Expanded(
            child: ListView(
              children: [
                Text(
                  'This 0',
                ),
                Text(
                  'This 1',
                ),
                Text(
                  'This 2',
                ),
                Text(
                  'This 3',
                ),
              ],
            ),
          ),
        ],
      ),
    );

```

Repro steps:
1. Run NVDA or any other screen reader on the windows machine.
2. Press tab to move the focus.

Actual: Not moving the focus to text and not reading it out.
Expected: Should move the focus to the text and read out loud.

**Comment by cbracken** [comment]
This is queued up after https://github.com/flutter/flutter/issues/109802.

**Comment by Hixie** [comment]
@yaakovschectman informs me this is currently blocked on them getting some information from an a11y expert.

**Comment by chunhtai** [comment]
we have shortcut that handles tab on the framework side, we should first make sure it does not swallow the key press.
Is this a a keyboard focus or a11y focus? If we really can't get it to work, is it possible to send a11y focus event to the next item when tab is pressed?

**Comment by yaakovschectman** [comment]
In reality, we have found TAB works in a manner consistent with other applications, but keyboard commands that should allow navigation when using a screen reader (e.g. INS+UP/DOWN for NVDA) do not. Additionally, I have found that when using a screen reader, the screen reader processes keystrokes before Flutter does, so issues of Flutter swallowing key presses seem implausible.

**Comment by cbracken** [comment]
Depending on how soon you expect to be able to talk to the person we discussed on Chromium, one other option is that NVDA is open source (appears to be written in python) so it may be possible to instrument it to get a better picture of what's going on.

**Comment by Hixie** [comment]
update: @yaakovschectman informs me that they are talking with someone from Chrome about this.

**Comment by cbracken** [comment]
Swapping assignment to myself while I attempt to clarify this bug and locate some win a11y experts who can help define the expected behaviour here. 

The current state is:
* NVDA [documents](https://dequeuniversity.com/screenreaders/nvda-keyboard-shortcuts) that the `Tab` key reads the next _focusable_ item such as (e.g. links, buttons). Text widgets are not focusable items and therefore our behaviour with regards to the `Tab` key is, I believe, working as intended.
* NVDA documents that the `down arrow` key reads the next item. When I reconstruct the same example as HTML in a web browser, the down arrow does read each item in turn. In Flutter, it does not. Therefore, I think there _is_ a bug to be resolved here. I'm following up with the internal filer on that side of things.

**Comment by jacobsimionato** [comment]
Just wondering if there have been any updates on this issue recently? I'm eager to unblock our Google customer!

**Comment by cbracken** [comment]
We're still investigating what's going on with the arrow key navigation. @yaakovschectman has confirmed that NVDA is getting the keypress before the Flutter embedder is. Given NVDA is open source, I'm having a look at whether I can debug from the NVDA side to reveal what's going on here.

**Comment by yaakovschectman** [comment]
I spoke with somebody from NVDA, and it turns out that the functionality of the arrow keys is not modified by NVDA screen reader. The arrow keys are handled by an app as they normally are, and the app may fire an event in response. [NVDA object navigation](https://www.nvaccess.org/files/nvda/documentation/userGuide.html#ObjectNavigation) can be used to navigate through non-focusable elements, which sounds like what this issue regards.
